src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -274,8 +300,9 @@ public void snapshotMixOfColocatedNonColocatedTables() throws Exception {
         assertEquals(recordCountInNonColocated, recordsForNonColocated.size());
     }
 
-    @Test
-    public void snapshotColocatedNonColocatedThenStream() throws Exception {
+    @ParameterizedTest
+    @ValueSource(booleans = {true,false})
+    public void snapshotColocatedNonColocatedThenStream(boolean initial_only) throws Exception {
```

Comment:
We follow camel-casing in Java, can you please see if `initial_only` can be changed to `initialOnly`